sdk/keyvault/Azure.ResourceManager.KeyVault/src/Generated/Operations.cs

```diff
@@ -41,9 +41,9 @@ internal Operations(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline,
 
         /// <summary> Lists all of the available Key Vault Rest API operations. </summary>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual AsyncPageable<Operation> ListAsync(CancellationToken cancellationToken = default)
+        public virtual AsyncPageable<Models.Operation> ListAsync(CancellationToken cancellationToken = default)
```

Comment:
Does this cause any issues for RPs that don't use the Models namespace?  Or does this just mean that everything needs to use Models namespace, or in those cases we plan to fully qualify the type?

I think this will present a problem in ARM with the new API design since Operation.cs won't be in Models we might need to think of a different name here.

Reply:
I think having types named `Operation` was already a problem even before this change highlighted the issue. It's confusing to have an `Operation<T>` in `Azure.Core` and 5 other `Operation` classes across libraries that are unrelated. 

For libraries that don't have a Models namespace and decide to still have a model called `Operation` customers would have to use part of the library namespace to disambiguate.

Reply:
Talked with @allenjzhang we are good with renaming this on our side since we have GAd